### ISSUE TITLE
Dependencies errors with beta7-12264

### ISSUE THREAD

**Issue by spaget** [aspnet/dnx#2318]
Hello, 

I got an error when i try to launch my project from Visual Studio (IIS Express or custom target). I'm on the last version of dnx (beta7-12264) since this morning.

Notes
* if i launch the project from command line it works fine 
```"Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://*:5001"```
* I have restore all packages

Here is the stacktrace: 
```
System.IO.FileLoadException: Impossible de charger le fichier ou l'assembly 'ProjectName' ou une de ses dépendances. Pointeur non valide (Exception de HRESULT : 0x80004003 (E_POINTER))
Nom de fichier : 'Evencity.Web.Admin' ---> System.ArgumentNullException: La valeur ne peut pas être null.
Nom du paramètre : type
   à Microsoft.Framework.Runtime.Common.DependencyInjection.ActivatorUtilities.CreateFactory(Type type)
   à Microsoft.Framework.Runtime.Common.DependencyInjection.ActivatorUtilities.CreateInstance(IServiceProvider services, Type type)
   à Microsoft.Framework.Runtime.CompilerServices.CreateService[T](IServiceProvider sp, IAssemblyLoadContext loadContext, TypeInformation typeInfo)
   à Microsoft.Framework.Runtime.ProjectLibraryExportProvider.b__5_1(TypeInformation typeInfo)
   à System.Collections.Generic.DictionaryExtensions.GetOrAdd[TKey,TValue](IDictionary`2 dictionary, TKey key, Func`2 factory)
   à Microsoft.Framework.Runtime.ProjectLibraryExportProvider.<>c__DisplayClass5_0.b__0(CacheContext ctx)
   à Microsoft.Framework.Runtime.Caching.CacheExtensions.<>c__DisplayClass0_0`1.b__0(CacheContext ctx)
   à Microsoft.Framework.Runtime.Caching.Cache.CreateEntry(Object k, Func`2 acquire)
   à Microsoft.Framework.Runtime.Caching.Cache.<>c__DisplayClass5_0.b__0()
   à System.Lazy`1.CreateValue()
   à System.Lazy`1.LazyInitValue()
   à System.Lazy`1.get_Value()
   à Microsoft.Framework.Runtime.Caching.Cache.Get(Object key, Func`2 factory)
   à Microsoft.Framework.Runtime.Caching.CacheExtensions.Get[T](ICache cache, Object key, Func`2 factory)
   à Microsoft.Framework.Runtime.ProjectLibraryExportProvider.GetLibraryExport(CompilationTarget target)
   à Microsoft.Framework.Runtime.CompositeLibraryExportProvider.<>c__DisplayClass2_0.b__0(ILibraryExportProvider r)
   à System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   à System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   à Microsoft.Framework.Runtime.CompositeLibraryExportProvider.GetLibraryExport(CompilationTarget target)
   à Microsoft.Framework.Runtime.LibraryManager.GetLibraryExport(String name, String aspect)
   à Microsoft.Framework.Runtime.Loader.ProjectAssemblyLoader.Load(AssemblyName assemblyName, IAssemblyLoadContext loadContext)
   à Microsoft.Framework.Runtime.Loader.ProjectAssemblyLoader.Load(AssemblyName assemblyName)
   à dnx.host.LoaderContainer.Load(AssemblyName assemblyName)
   à dnx.host.DefaultLoadContext.LoadAssembly(AssemblyName assemblyName)
   à Microsoft.Framework.Runtime.Loader.AssemblyLoaderCache.GetOrAdd(AssemblyName name, Func`2 factory)
   à Microsoft.Framework.Runtime.Loader.LoadContext.LoadAssemblyImpl(AssemblyName assemblyName)
   à Microsoft.Framework.Runtime.Loader.LoadContext.TryLoadAssembly(LoadContext context, AssemblyName assemblyName, Assembly& assembly)
   à Microsoft.Framework.Runtime.Loader.LoadContext.ResolveAssembly(Object sender, ResolveEventArgs args)
   à System.AppDomain.OnAssemblyResolveEvent(RuntimeAssembly assembly, String assemblyFullName)
   à System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   à System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   à System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   à System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   à Microsoft.AspNet.Hosting.Startup.StartupLoader.FindStartupType(String startupAssemblyName, IList`1 diagnosticMessages)
   à Microsoft.AspNet.Hosting.Internal.HostingEngine.EnsureStartup()
   à Microsoft.AspNet.Hosting.Internal.HostingEngine.EnsureApplicationServices()
   à Microsoft.AspNet.Hosting.Internal.HostingEngine.get_ApplicationServices()
   à Microsoft.AspNet.Loader.IIS.RuntimeHttpApplication.ApplicationStart(IHttpApplication application)
   à Microsoft.AspNet.Loader.IIS.HttpApplicationBase.InvokeApplicationStart(IHttpApplication application)
```

Any idea ?

Thanks !

**Comment by Grinderofl** [comment]
I'm experiencing a similar issue, but with console. Visual Studio refuses to launch any projects, console or web, while using beta7-12264. I started experiencing it yesterday, but seeing as there was a lot of renames going on, I decided to wait a little while until the other packages got updated. I've cleared out my package cache as well.

```
{"Exception has been thrown by the target of an invocation."}

*Trace*
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at dnx.clr.managed.RuntimeBootstrapper.Execute(String[] argv, FrameworkName targetFramework)
   at DomainManager.Main(Int32 argc, String[] argv)
```

=> InnerException:
```
{"Could not load file or assembly 'ConsoleSample' or one of its dependencies. Invalid pointer (Exception from HRESULT: 0x80004003 (E_POINTER))":"ConsoleSample"}

*Trace*
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.Framework.Runtime.Loader.LoadContext.Load(AssemblyName assemblyName)
   at Microsoft.Framework.Runtime.AssemblyLoadContextExtensions.Load(IAssemblyLoadContext loadContext, String name)
   at Microsoft.Framework.Runtime.DefaultHost.GetEntryPoint(String applicationName)
   at Microsoft.Framework.ApplicationHost.Program.ExecuteMain(DefaultHost host, String applicationName, String[] args)
   at Microsoft.Framework.ApplicationHost.Program.Main(String[] args)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.Framework.Runtime.Common.EntryPointExecutor.Execute(Assembly assembly, String[] args, IServiceProvider serviceProvider)
   at dnx.host.Bootstrapper.RunAsync(List`1 args, IRuntimeEnvironment env, FrameworkName targetFramework)
   at dnx.host.RuntimeBootstrapper.ExecuteAsync(String[] args, FrameworkName targetFramework)
   at dnx.host.RuntimeBootstrapper.Execute(String[] args, FrameworkName targetFramework)
```

=> InnerException
```
{"Value cannot be null.\r\nParameter name: type"}

*Trace*

   at Microsoft.Framework.Runtime.Common.DependencyInjection.ActivatorUtilities.CreateFactory(Type type)
   at Microsoft.Framework.Runtime.Common.DependencyInjection.ActivatorUtilities.CreateInstance(IServiceProvider services, Type type)
   at Microsoft.Framework.Runtime.CompilerServices.CreateService[T](IServiceProvider sp, IAssemblyLoadContext loadContext, TypeInformation typeInfo)
   at Microsoft.Framework.Runtime.ProjectLibraryExportProvider.<GetLibraryExport>b__5_1(TypeInformation typeInfo)
   at System.Collections.Generic.DictionaryExtensions.GetOrAdd[TKey,TValue](IDictionary`2 dictionary, TKey key, Func`2 factory)
   at Microsoft.Framework.Runtime.ProjectLibraryExportProvider.<>c__DisplayClass5_0.<GetLibraryExport>b__0(CacheContext ctx)
   at Microsoft.Framework.Runtime.Caching.CacheExtensions.<>c__DisplayClass0_0`1.<Get>b__0(CacheContext ctx)
   at Microsoft.Framework.Runtime.Caching.Cache.CreateEntry(Object k, Func`2 acquire)
   at Microsoft.Framework.Runtime.Caching.Cache.<>c__DisplayClass5_0.<AddEntry>b__0()
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.LazyInitValue()
   at System.Lazy`1.get_Value()
   at Microsoft.Framework.Runtime.Caching.Cache.Get(Object key, Func`2 factory)
   at Microsoft.Framework.Runtime.Caching.CacheExtensions.Get[T](ICache cache, Object key, Func`2 factory)
   at Microsoft.Framework.Runtime.ProjectLibraryExportProvider.GetLibraryExport(CompilationTarget target)
   at Microsoft.Framework.Runtime.CompositeLibraryExportProvider.<>c__DisplayClass2_0.<GetLibraryExport>b__0(ILibraryExportProvider r)
   at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Microsoft.Framework.Runtime.CompositeLibraryExportProvider.GetLibraryExport(CompilationTarget target)
   at Microsoft.Framework.Runtime.LibraryManager.GetLibraryExport(String name, String aspect)
   at Microsoft.Framework.Runtime.Loader.ProjectAssemblyLoader.Load(AssemblyName assemblyName, IAssemblyLoadContext loadContext)
   at Microsoft.Framework.Runtime.Loader.ProjectAssemblyLoader.Load(AssemblyName assemblyName)
   at dnx.host.LoaderContainer.Load(AssemblyName assemblyName)
   at dnx.host.DefaultLoadContext.LoadAssembly(AssemblyName assemblyName)
   at Microsoft.Framework.Runtime.Loader.AssemblyLoaderCache.GetOrAdd(AssemblyName name, Func`2 factory)
   at Microsoft.Framework.Runtime.Loader.LoadContext.LoadAssemblyImpl(AssemblyName assemblyName)
   at Microsoft.Framework.Runtime.Loader.LoadContext.ResolveAssembly(Object sender, ResolveEventArgs args)
   at System.AppDomain.OnAssemblyResolveEvent(RuntimeAssembly assembly, String assemblyFullName)
```

**Comment by evorine** [comment]
I have same problem here.
May be this is related with https://github.com/aspnet/Announcements/issues/45.

**Comment by BrennanConroy** [comment]
Fixed in dbbac90b850ac5f329167e89c3a88b8e868b22f8
Use a different dnx version for now or wait until a new build is out.
